web/app/scripts/views/map/layers-controller.js

```diff
@@ -771,7 +781,7 @@
         }
 
         function getTilekeys() {
-            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams()).then(
+            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams(), true, true, false).then(
```

Comment:
Here it should be just getting the tile key and not the actual records. I set this one for all fields so that Django would have what it needs to apply the filters.